src/integrationtests/java/com/aws/greengrass/integrationtests/SyncTest.java

```diff
@@ -353,7 +360,7 @@ void GIVEN_sync_config_and_no_cloud_WHEN_startup_THEN_cloud_version_updated_via_
                 .mockCloud(true)
                 .mockDao(true)
                 .build());
-
+        cdl.await(2,TimeUnit.SECONDS);
```

Comment:
this doesn't validate that we successfully counted down. Use assertTrue to verify that.